zerver/lib/events.py

```diff
@@ -902,6 +903,9 @@ def fetch_initial_state_data(
         # abuse.
         state["giphy_api_key"] = settings.GIPHY_API_KEY if settings.GIPHY_API_KEY else ""
 
+    if want("push_account"):
```

Comment:
Did we discuss whether this should be singular or plural?

Reply:
CZO Discussion - [#api design > E2EE - register-queue @ 💬](https://chat.zulip.org/#narrow/channel/378-api-design/topic/E2EE.20-.20register-queue/near/2214628)

Decision: Singular